css/style.css

```
@@ -16,4 +16,129 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* This is Header (Navigation & Hero )*/
+
+/* Navigation */
+nav{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:140px;
+    width: 80%;
+    margin: auto;
+    font-size: 2.5rem;
+    }
+
+a{
+    height: 60px;
+    margin-left: 80px;
+    text-decoration: none;
+    color: rgb(65, 63, 63);
+    }
+
+a:hover{
```

```suggestion
a:focus,
a:hover{
```
When we add hover, we should also add focus so keyboard users who use the `Tab` key to jump to this link will get these styles too